### ISSUE TITLE
revert #16

### ISSUE THREAD

**Issue by ulbqb** [Finschia/cosmy-wasmy#18]

**Comment by 170210** [comment]
@ulbqb Yes, but since cosmy-wasmy is already using @cosmjs/amino, which contains cosmjs/crypto, it will trigger a warning if not modified.

**Comment by ulbqb** [comment]
The cause of the warning it use different versions between our `@cosmjs/crypto` and `@cosmjs/amino`'s one.

```javascript
import { HdPath,Slip10RawIndex } from "@cosmjs/amino/node_modules/@cosmjs/crypto"; 
```
I don't think this is correct usage of package.
